micrometer/src/main/scala/zio/metrics/connectors/micrometer/package.scala

```diff
@@ -9,7 +9,7 @@ import io.micrometer.core.instrument.{MeterRegistry, Tag}
 
 package object micrometer {
 
-  lazy val micrometerLayer: ZLayer[MeterRegistry, Nothing, Unit] =
+  val micrometerLayer: ZLayer[MeterRegistry with MicroMeterConfig, Nothing, Unit] =
```

Comment:
why do you prefer to remove laziness here? I think it is ok to have lazy val layer

Reply:
Layers are already lazy by default

Reply:
yes cause it is effect :) but constructing effect also need some allocation. Approach with lazy layer mensioned in ZIO documentation (https://zio.dev/reference/di/manual-layer-construction/). And similarly is used in other modules:
- https://github.com/zio/zio-metrics-connectors/blob/zio/series2.x/core/src/main/scala/zio/metrics/connectors/datadog/package.scala#L11
- https://github.com/zio/zio-metrics-connectors/blob/zio/series2.x/core/src/main/scala/zio/metrics/connectors/newrelic/package.scala#L9
- https://github.com/zio/zio-metrics-connectors/blob/zio/series2.x/core/src/main/scala/zio/metrics/connectors/prometheus/package.scala#L8
- https://github.com/zio/zio-metrics-connectors/blob/zio/series2.x/core/src/main/scala/zio/metrics/connectors/statsd/package.scala#L8

Thats why I decided to make layer lazy. On the other hand this is the single layer in package in separated module. So if user uses it as a dependency that definitely he will use this layer.

Reply:
I've added it back